src/js/views/admin/program/flow/program-flow.scss

```diff
@@ -133,6 +133,37 @@
   vertical-align: middle;
 }
 
+.program-flow__sort-handle {
+  align-items: center;
+  bottom: -16px;
+  cursor: move;
+  display: none;
+  justify-content: center;
+  position: absolute;
+  right: -16px;
+  text-align: center;
+  top: -16px;
+  width: 60px;
+}
+
+.program-flow__action-item:hover {
+  .program-flow__sort-handle {
+    display: flex;
+  }
+
+  .program-flow__action-ts {
+    display: none;
+  }
+}
+
+.is-dragging {
+  .program-flow__action-item:not(.sortable-ghost) {
+    .program-flow__sort-handle {
```

Comment:
I moved most of the dragging styles into table-list.scss. Opted to leave the handle display styles in the program-flow in case other lists don't have the handle displayed on hover. But it should be doable to move all the handle styles into table-list.scss and use a generic `sort--handle` classname if that's appropriate.